ui/src/views/user/settings/SettingsView.vue

```diff
@@ -4,78 +4,181 @@
+              />
+            </v-col>
+          </v-row>
+          <ValidationObserver v-slot="{ handleSubmit }">
+            <v-form
+              v-model="isFormValid"
+              class="user-settings-form"
+              @keyup.native.enter="handleSubmit(onFormSubmit)"
+              @submit.prevent="handleSubmit(onFormSubmit)"
+            >
+              <v-container class="py-0">
+                <v-row>
+                  <v-col
+                    cols="12"
+                    class="my-0 py-0"
+                  >
+                    <ValidationProvider
+                        v-slot="{ errors }"
+                        name="E-mail"
+                        rules="required|email"
+                    >
+                      <v-text-field
+                        v-model.trim="userEmail"
+                        outlined
+                        required
+                        disabled
+                        :error-messages="errors[0]"
+                        label="E-mail"
+                        class="my-0 py-0"
+                      />
+                    </ValidationProvider>
+                  </v-col>
+                  <v-col
+                    cols="12"
+                    class="my-0 py-0"
+                  >
+                    <ValidationProvider
+                        v-slot="{ errors }"
+                        name="Password"
+                        rules="required|min:8"
+                    >
+                      <v-text-field
+                        v-model.trim="formData.password"
+                        outlined
+                        required
+                        hint="Minimum 8 characters"
+                        persistent-hint
+                        label="Password"
+                        class="my-0 py-0"
+                        :append-icon="isPasswordShown ? 'mdi-eye' : 'mdi-eye-off'"
+                        :type="isPasswordShown ? 'text' : 'password'"
+                        :error-messages="errors[0]"
+                        @click:append="isPasswordShown = !isPasswordShown"
+                      />
+                    </ValidationProvider>
+                  </v-col>
+                  <v-col
+                    cols="12"
+                  >
+                    <v-row justify="center">
+                      <v-btn
+                        color="primary"
+                        min-width="150"
+                        class="mt-2 mb-5"
+                        v-bind:disabled='!isDisabled'
+                        @click="handleSubmit(onFormSubmit)"
+                      >
+                        SAVE SETTINGS
+                      </v-btn>
+                    </v-row>
+                    <v-row justify="center">
+                      <v-btn
+                        text
+                        min-width="150"
+                        class="text-none pa-0 mt-2 mb-5"
+                        @click="onDeleteAccount()"
```

Comment:
This text should be underlined and the text formatting should be removed. I will write a general class so that we can use the class later; however for now please use the styling I wrote earlier.